README.md

```diff
@@ -6,13 +6,30 @@
 
 
 ## Usage
-
-> From root directory:
-
-> Install dependencies: npm install
-
-
-> Seed mongodb database: npm run db:setup
+> To begin install dependencies: npm install
+>
+> To load seeded database, you will need PostgreSQL installed on your machine.
+>
+> Generate data to a csv file with the following command in the terminal from SimilarProducts root folder : npm run generate
+>
+> Then follow these steps in the psql terminal:
+
+>   1. Make db:
+> Copy this into the terminal and enter:
+>
+> CREATE DATABASE similar_prod;
+>
+>   2. Make table:
+> Copy this into the terminal and enter:
+>
+> CREATE TABLE products (productId INT PRIMARY KEY, name VARCHAR(200) NOT NULL, category VARCHAR(200) NOT NULL, price REAL NOT NULL, rating REAL, imageurl VARCHAR(200), onSale BOOLEAN DEFAULT false);
+>
+>   3. Seed table:
+> Copy this into the terminal and enter:
+>
+> COPY products(productid, name, category, price, rating, imageurl, onSale) from '/Users/clairemelbourne/Desktop/ProjectPhase/SDC/SimilarProducts/data/primary-records.csv' DELIMITER ',' CSV HEADER;
+
+To activate development version of site:
```

Comment:
Very clear. nice job

Maybe add a date at the top, so if a developer changes something people will have a reference to if the Readme was updated with the alterations.